frontend/src/Settings/General/GeneralSettings.js

```diff
@@ -183,12 +186,10 @@ class GeneralSettings extends Component {
         <ConfirmModal
           isOpen={this.state.isRestartRequiredModalOpen}
           kind={kinds.DANGER}
-          title="Restart Sonarr"
-          message={
-            `Sonarr requires a restart to apply changes, do you want to restart now? ${isWindowsService ? 'Depending which user is running the Sonarr service you may need to restart Sonarr as admin once before the service will start automatically.' : ''}`
-          }
-          cancelLabel="I'll restart later"
-          confirmLabel="Restart Now"
+          title={translate('RestartSonarr')}
+          message={`${translate('RestartRequiredToApplyChanges')} ${isWindowsService ? translate('RestartRequiredToApplyChangesIsWindowsService') : ''}`}
```

Comment:
`RestartRequiredToApplyChangesIsWindowsService` this also seems needlessly verbose